lib/core/form/components/form-base.component.ts

```diff
@@ -48,6 +48,10 @@ export abstract class FormBaseComponent {
     @Input()
     disableCompleteButton: boolean = false;
 
+    /** If true then the `Save` outcome button is shown but it will be disabled. */
+    @Input()
+    disableSaveButton: boolean = false;
```

Comment:
I we had this property here we need to modify also the documentation of
adf-form
adf-form-cloud